### ISSUE TITLE
[Feature] Auto pass/fail CI if run against the same sha

### ISSUE THREAD

**Issue by erikogan** [jaredatron/hobson#12]
If I push a hobson build of a sha, and then I merge to a branch that CI is running against, it should just point to that build that already ran.

**Comment by jaredatron** [comment]
I dont think we want this given the possible existence of intermittently failing tests. Maybe just auto-pass a sha that has passed before?
